content/v2.0/write-data/_index.md

```diff
@@ -234,6 +236,62 @@ mem,host=host2 used_percent=27.18294630 1556896336
 _For information about **InfluxDB API response codes**, see
 [InfluxDB API Write documentation](/v2.0/api/#operation/PostWrite)._
 
+### Third-party technologies
```

Comment:
I think this needs some embellishment....

There are a number of third-party technologies which can be configured to work with InfluxDB directly.  Essentially, these technologies can be configured to send line protocol directly to InfluxDB. If you are using any of the following technologies, here are some handy links describing how you can configure them to write data to InfluxDB (**no additional software to download or install**)